tests/utils.py

```diff
@@ -11,3 +20,43 @@ def func(*args):
             return DEFAULT
 
     return func
+
+
+def file_row_harness(row: List[Union[str, int]], layout: dict):
+    """ Write row to file for testing in ingest """
+    lay = Layout(layout, no_header=True)
+
+    with tempfile.TemporaryDirectory() as d:
+        csv_p = Path(d, 'file_test.csv')
+
+        with csv_p.open('w', newline='') as f:
+            writer = csv.writer(f)
+            writer.writerow(row)
+
+        xl_p = Path(d, 'excel_test.xlsx')
+        wb = openpyxl.Workbook()
+        sheet = wb['Sheet']
+        for ix, value in enumerate(row, start=1):
+            sheet.cell(row=1, column=ix).value = value
+        wb.save(filename=xl_p)
+
+        to_check = [
+            ('CsvFile', CsvFile(lay), csv_p),
+            ('ExcelFile', ExcelFile(lay), xl_p)
+        ]
+        aes = {}
+        for nm, obj, p in to_check:
+            try:
+                assert not obj.check(p)
+            except AssertionError as e:
+                aes[nm] = e
+        new_line = '\n'
+        assert not aes, f'Write test failed for:\n {new_line.join([f"{k}:{v}" for k, v in aes.items()])}'
+
+
+def file_cell_harness(value: Union[str, list], field: Union[Field, dict]):
+    """ Wrapper to convert cell to row for harness check """
+    if isinstance(value, list):
```

Comment:
I think this change makes file_cell wrapper equivalent to file_row_harness. Let's try replacing any time this is called for a row with `file_row_harness` instead. If it doesn't blow up, we should revert `file_cell_harness` back to only taking a single value.

Reply:
Yuuup, you're absolutely right. Reverted `file_cell_harness` and used `file_row_harness` instead and it just works. Smh